boards/arm/wio_terminal/wio_terminal.dts

```diff
@@ -0,0 +1,271 @@
+			reg = <0x00000000 DT_SIZE_K(16)>;
+			read-only;
+		};
+
+		code_partition: partition@4000 {
+			label = "code";
+			reg = <0x4000 DT_SIZE_K(512-16)>;
+			read-only;
+		};
+	};
+};
+
+&sercom0 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <0>;
+	pinctrl-0 = <&sercom0_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portc 25 GPIO_ACTIVE_LOW>;
+};
+
+&sercom1 {
+	status = "okay";
+	compatible = "atmel,sam0-uart";
+	current-speed = <115200>;
+	rxpo = <1>;
+	txpo = <0>;
+	pinctrl-0 = <&sercom1_uart_default>;
+	pinctrl-names = "default";
+};
+
+&sercom2 {
+	status = "okay";
+	compatible = "atmel,sam0-uart";
+	current-speed = <115200>;
+	rxpo = <1>;
+	txpo = <0>;
+	pinctrl-0 = <&sercom2_uart_default>;
+	pinctrl-names = "default";
+};
+
+&sercom3 {
+	status = "okay";
+	compatible = "atmel,sam0-i2c";
+	clock-frequency = <I2C_BITRATE_FAST>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+	pinctrl-0 = <&sercom3_i2c_default>;
+	pinctrl-names = "default";
+};
+
+&sercom4 {
+	status = "okay";
+	compatible = "atmel,sam0-i2c";
+	clock-frequency = <I2C_BITRATE_HIGH>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+	pinctrl-0 = <&sercom4_i2c_default>;
+	pinctrl-names = "default";
+
+	/* LIS3DH */
+	lis3dh: lis3dh@18 {
+		compatible = "st,lis3dh", "st,lis2dh";
+		status = "okay";
+		reg = <0x18>;
+		irq-gpios = <&portc 21 GPIO_ACTIVE_HIGH>;
+	};
+
+	/* ATECC608A */
+	atecc608a@6a {
+		compatible = "atmel,atecc608";
+		reg = <0x6a>;
+	};
+};
+
+&sercom5 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <0>;
+	pinctrl-0 = <&sercom5_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portb 1 GPIO_ACTIVE_LOW>;
+};
+
+&sercom6 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <0>;
+	pinctrl-0 = <&sercom6_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portc 19 GPIO_ACTIVE_LOW>;
+
+  /* MicroSD Card */
+	sdhc0: sdhc@0 {
+		compatible = "zephyr,sdhc-spi-slot";
+		reg = <0>;
+		spi-max-frequency = <24000000>;
+		mmc {
+			compatible = "zephyr,sdmmc-disk";
+		};
+	};
+};
+
+&sercom7 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <3>;
+	pinctrl-0 = <&sercom7_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portb 21 GPIO_ACTIVE_LOW>;
+
+	/* LCD */
+	ili9341: ili9341@0 {
+		compatible = "ilitek,ili9341";
+		spi-max-frequency = <24000000>;
+		reg = <0>;
+		cmd-data-gpios = <&portc 6 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&portc 7 GPIO_ACTIVE_LOW>;
+		pixel-format = <ILI9XXX_PIXEL_FORMAT_RGB565>;
+		rotation = <270>;
+		width = <240>;
+		height = <320>;
+	};
+};
+
+zephyr_udc0: &usb0 {
+	status = "okay";
+	pinctrl-0 = <&usb_dc_default>;
+	pinctrl-names = "default";
+
+	cdc_acm_uart0: cdc_acm_uart0 {
+		compatible = "zephyr,cdc-acm-uart";
+	};
```

Comment:
Do not use generic names here. It is better use a prefix like `wio_terminal_console`

Reply:
Done